ecosystem/indexer-grpc/indexer-grpc-fullnode/src/convert.rs

```diff
@@ -171,20 +171,9 @@ pub fn convert_transaction_payload(
             )),
         },
 
-        // Deprecated. Will be removed in the future.
-        TransactionPayload::ModuleBundlePayload(mbp) => transaction::TransactionPayload {
-            r#type: transaction::transaction_payload::Type::ModuleBundlePayload as i32,
-            payload: Some(
-                transaction::transaction_payload::Payload::ModuleBundlePayload(
-                    transaction::ModuleBundlePayload {
-                        modules: mbp
-                            .modules
-                            .iter()
-                            .map(convert_move_module_bytecode)
-                            .collect(),
-                    },
-                ),
-            ),
+        // Deprecated.
+        TransactionPayload::ModuleBundlePayload(_) => {
+            unreachable!("Module bundle payload has been removed")
```

Comment:
@bowenyang007 is this unreachable ok here? All interfaces return concrete types and not options/results, so it is not easy to do something else here.

Reply:
@bowenyang007 ?